webapps/docs/ssi-howto.xml

```diff
@@ -146,27 +146,53 @@ false.</li>
 <p>The directives are:</p>
 <ul>
 <li>
-<strong>config</strong> - <code>&lt;!--#config timefmt=&quot;%B %Y&quot; --&gt;</code>
-Used to set the format of dates and other items processed by SSI
+<strong>config</strong> - <code>&lt;!--#config errmsg=&quot;Error occured&quot; sizefmt=&quot;abbrev&quot; timefmt=&quot;%B %Y&quot; --&gt;</code>
+Used to set SSI error message, the format of dates and file sizes processed by SSI.<br/>
+All are optional but at least one must be used. The available options are as follows:
+<br/>
+<strong>errmsg</strong> - error message used for SSI errors<br/>
+<strong>sizefmt</strong> - format used for sizes in the <strong>fsize</strong> directive<br/>
+<strong>timefmt</strong> - format used for timestamps in the <strong>flastmod</strong> directive<br/>
 </li>
 <li>
-<strong>echo</strong> -   <code>&lt;!--#echo var=&quot;VARIABLE_NAME&quot; --&gt;</code>
+<strong>echo</strong> -   <code>&lt;!--#echo var=&quot;VARIABLE_NAME&quot; encoding=&quot;entity&quot; --&gt;</code>
 will be replaced by the value of the variable.
+<br/>
+The optional <strong>encoding</strong> parameter specifies the type of encoding to use.
+Valid values are <strong>entity</strong> (default), <strong>url</strong> or <strong>none</strong>.
+NOTE: Using an encoding other than <strong>entity</strong> can lead to security issues.
 </li>
 <li>
-<strong>exec</strong> -  Used to run commands on the host system.
+<strong>exec</strong> - <code>&lt;!--#exec cmd=&quot;file-name&quot; --&gt;</code>
+Used to run commands on the host system.
+</li>
+<li>
+<strong>exec</strong> - <code>&lt;!--#exec cgi=&quot;file-name&quot; --&gt;</code>
+This acts the same as the <strong>include</strong> directive, and doesn't actually execute any commands.
+</li>
+<li>
+<strong>include</strong> -  <code>&lt;!--#include file=&quot;file-name&quot; --&gt;</code>
+inserts the contents, this is interpreted relative to the current directory.
 </li>
 <li>
 <strong>include</strong> -  <code>&lt;!--#include virtual=&quot;file-name&quot; --&gt;</code>
-inserts the contents
+inserts the contents, this is interpreted relative to the document being served.
 </li>
 <li>
 <strong>flastmod</strong> - <code>&lt;!--#flastmod file=&quot;filename.shtml&quot; --&gt;</code>
-Returns the time that a file was lost modified.
+Returns the time that a file was lost modified, this is interpreted relative to the current directory.
+</li>
```

Comment:
"last" not "lost"
"relative to the current directory"?

Reply:
Meaning its not a virtual path, I change the language to clarify